### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by alex-3356** [noahzn/Lite-Mono#195]
Sorry to bother you.  I would like to ask if the dataset used for training is the same for each epoch?

**Comment by noahzn** [comment]
Hi, I don't get you. "For each epoch", if you mean the batch used in each epoch, they may be different because we set `drop_last=True` in the data loader.

**Comment by alex-3356** [comment]
Your --num_epochs is set to 30. I would like to ask if each epoch uses the same set of images for training?

**Comment by noahzn** [comment]
It should be the same. We used `drop_last=True` but we didn't use `shuffle=True`. If you want the same to be exactly the same also for each training, you can set a seed.

**Comment by alex-3356** [comment]
I would like to ask why the same images are used for training in every epoch.

**Comment by noahzn** [comment]
Each epoch uses all the images for training. Why do you think they are different?

**Comment by alex-3356** [comment]
No, I'm just curious.
